WordPress/src/main/res/values/strings.xml

```diff
@@ -111,6 +111,7 @@
     <string name="site_cannot_be_loaded">We cannot load the data for your site right now. Please try again later</string>
     <string name="local_draft">Local draft</string>
     <string name="local_changes">Local changes</string>
+    <string name="publish_now">Publish now</string>
```

Comment:
We might want to use capital "N" in "Now". Wdyt?

Reply:
Whoops! Thanks f127baa